testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -416,6 +416,65 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>app-server-jetty81</id>
+            <activation>
+                <property>
+                    <name>app.server</name>
+                    <value>jetty</value>
```

Comment:
Do we plan to test all jetty versions in one test run? If not there should me imho jetty81, shouldn't be?

Reply:
This is very good question...

All Jetty implementation share the same code. What changes between them, are dependencies. So far I've been using `@AppServerContainer(ContainerConstants.APP_SERVER_JETTY)`, which means the testing code doesn't differentiate Jetty versions. So far this works fine, so I don't see any reason to introduce specific versions in Jetty App Server. I hope this won't change between Jetty 9.4, 9.3 and 9.2 implementations (as 9.3 and 9.2 are not there yet). 

So unless there's any good reason, perhaps we could stay with `app.server=jetty` without any version? 

@mhajas @hmlnarik @vramik WDYT?

Reply:
@slaskawi thanks. 

If you stay with `app.server=jetty` without any version, how do you know which version of jetty server should be started? Or do you plan to test all version in one test run? In that case you'd need to rerun every single test for every jetty version. Something like `@ParameterizedTest` introduced in JUnit 5? Maybe I'm missing something. 

I'll do the review again with you latest changes and see if I can understand it better. Is it ready for the review now?

Reply:
APP_SERVER_ annotations should correspond to available app server modules, i.e. should differentiate between the version. APP_SERVER_JETTY81, APP_SERVER_JETTY92, APP_SERVER_JETTY93, APP_SERVER_JETTY94. Cf. [APP_SERVER_WILDFLY](https://github.com/keycloak/keycloak/blob/24b7d080af6ea554e346aa4ab770f00450696e4d/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/ContainerConstants.java#L24) and [APP_SERVER_WILDFLY_DEPRECATED](https://github.com/keycloak/keycloak/blob/24b7d080af6ea554e346aa4ab770f00450696e4d/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/ContainerConstants.java#L25):

Reply:
Ok, great! Thanks for the explanation. I'll try to introduce necessary changes shortly.

Reply:
Implemented thanks!